src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
`m`, as well as `f` (which is used in `calculateWomenAverageAge`) can be extraxred to separate enum, which will give you more meaningful naming and also in case some day data will change from `m` to `male` you won't need to refactor all places and you can just update single enum:

```js
const Sex = {
  Male: 'm',
  Female: 'f',
}
```

```suggestion
    ? people.filter((p) => p.sex === Sex.Male && Math.ceil(p.died / 100) === century)
```